platform-includes/replay/privacy-configuration/javascript.mdx

```diff
@@ -3,6 +3,7 @@ you can opt out of the default text masking and image blocking by configuring th
 
 ```javascript
 Sentry.replayIntegration({
+  // NOTE: This will disable built-in masking. Look at other options opt-in to masking sensitive data
```

Comment:
```suggestion
  // NOTE: This will disable built-in masking. Only use this if you site has no sensitive data or, paired with other options for blocking parts of the site with sensitive data such as 'ignore', 'block', 'mask' and 'maskFn'
```